src/components/Sidebar.tsx

```diff
@@ -0,0 +1,54 @@
+import { FiUsers, FiLogOut, FiFileText } from 'react-icons/fi';
+
+const Sidebar = () => {
+    return (
+        <div 
+            className="w-16 h-screen flex flex-col justify-between py-4"
+            style={{ backgroundColor: '#002D7E' }}
```

Comment:
Don't use style to stle background colors, opt to tailwind CSS colors. You can see in @Satvik26 login's page how to use our pallete colors